Qt5 builds failing on Bitrise — missing /usr/local/.//mkspecs/macx-clang

Qt5 macOS builds have been failing on Bitrise for the past few days, as seen in https://www.bitrise.io/build/243eaa19e4ba6066 (#6601) and https://www.bitrise.io/build/bda5e310bc1dedf1 (#6709):

```
-- [Mason] Unpacking package to mason_packages/headers/geojson/0.3.2...
CMake Error at /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:15 (message):
  The imported target "Qt5::Core" references the file

     "/usr/local/.//mkspecs/macx-clang"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "/usr/local/lib/cmake/Qt5Core/Qt5CoreConfigExtras.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  /usr/local/lib/cmake/Qt5Core/Qt5CoreConfigExtras.cmake:50 (_qt5_Core_check_file_exists)
  /usr/local/lib/cmake/Qt5Core/Qt5CoreConfig.cmake:141 (include)
  platform/qt/qt5.cmake:1 (find_package)
  platform/qt/qt.cmake:72 (include)
  platform/qt/config.cmake:1 (include)
  CMakeLists.txt:51 (include)


-- Configuring incomplete, errors occurred!
See also "/Users/vagrant/git/build/qt-macos-x86_64/Debug/CMakeFiles/CMakeOutput.log".
make: *** [build/qt-macos-x86_64/Debug/build.ninja] Error 1
```

/cc @brunoabinader @tmpsantos

--- a/platform/qt/bitrise-qt5.yml
+++ b/platform/qt/bitrise-qt5.yml
@@ -34,8 +34,9 @@ workflows:
             brew install qt5
             brew link qt5 --force
             brew linkapps qt5
-            ln -s /usr/local/Cellar/qt5/5.6.1-1/mkspecs /usr/local/mkspecs
-            ln -s /usr/local/Cellar/qt5/5.6.1-1/plugins /usr/local/plugins
+            export HOMEBREW_QT5_VERSION=$(brew list --versions qt5 | rev | cut -d' ' -f1 | rev)
+            ln -s /usr/local/Cellar/qt5/$HOMEBREW_QT5_VERSION/mkspecs /usr/local/mkspecs
+            ln -s /usr/local/Cellar/qt5/$HOMEBREW_QT5_VERSION/plugins /usr/local/plugins
             export BUILDTYPE=Debug
             make qt-app
             make qt-qml-app

